staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder/builder.go

```diff
@@ -84,8 +103,7 @@ type Options struct {
 	AllowNonStructural bool
 }
 
-// BuildSwagger builds swagger for the given crd in the given version
-func BuildSwagger(crd *apiextensionsv1.CustomResourceDefinition, version string, opts Options) (*spec.Swagger, error) {
+func generateBuilder(crd *apiextensionsv1.CustomResourceDefinition, version string, opts Options) (*builder, error) {
```

Comment:
a builder for a builder 🤷

Reply:
Pray we don't alter it further with `GenericInstanceHandlerBuilderBuilder`